### PR TITLE
feat: worker displaying in section show

### LINKED ISSUE #135: feat: display affected worker in section show
Same is title

### PR DESCRIPTION
**Pull request by vfricou**
Close #135

### CODE REVIEW

**Comment by sebastiendeu** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There're changes, but no changelog.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/views/sections/show.html.haml [hunk 1]
```diff
@@ -21,6 +21,34 @@
...
+        %span.info-box-number
+          = '%{name} (VID : %{vid})' % { name: @section.vlan_name, vid: @section.vlan_vid }
+  %div{class: last_scan == 0 ? 'col-6' : 'col-4' }
+    %div.info-box.mt-4{class: dark_mode_class}
+      %div.info-box-content
+        %spam.info-box-text
+          = _('Affected worker :')
+        %span.info-box-number
+          - unless @section.worker.nil?
+            = '%{name}' % { name: @section.worker_name }
```
Comment (sebastiendeu) [inline app/views/sections/show.html.haml:39]:
```suggestion
            = @section.worker_name
```

File: app/views/sections/show.html.haml [hunk 2]
```diff
@@ -21,6 +21,34 @@
...
+            = _('No worker assigned')
+
+  - unless last_scan == 0
+    %div.col-4
+      %div.info-box.mt-4{class: dark_mode_class}
+        %div.info-box-content
+          %span.info-box-text
+            = _('Last scan time :')
+          %span.info-box-number
+            = '%{timestamp}' % { timestamp: Time.at(last_scan) }
```
Comment (sebastiendeu) [inline app/views/sections/show.html.haml:50]:
```suggestion
            = Time.at(last_scan)
```

**Review by sebastiendeu — CHANGES_REQUESTED** [review]

File: app/views/sections/show.html.haml (hunk 1 above)
Comment (vfricou) [reply app/views/sections/show.html.haml]: Fixed in latest commit

File: app/views/sections/show.html.haml (hunk 2 above)
Comment (vfricou) [reply app/views/sections/show.html.haml]: Fixed in latest commit

**Review by sebastiendeu — APPROVED** [review]

**Pull request merged**